src/common/transformations/include/transformations/dimension_tracking.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <openvino/pass/graph_rewrite.hpp>
+#include <openvino/opsets/opset1.hpp>
+
+using namespace std;
```

Comment:
suggesting remove this, it is always recommended avoid 'using namespace' in header files